gensrc/proto/cloud.proto

```diff
@@ -1550,6 +1562,8 @@ service MetaService {
     rpc update_delete_bitmap(UpdateDeleteBitmapRequest) returns(UpdateDeleteBitmapResponse);
     rpc get_delete_bitmap(GetDeleteBitmapRequest) returns(GetDeleteBitmapResponse);
     rpc get_delete_bitmap_update_lock(GetDeleteBitmapUpdateLockRequest) returns(GetDeleteBitmapUpdateLockResponse);
+    rpc update_delete_bitmap_without_lock(UpdateDeleteBitmapRequest) returns(UpdateDeleteBitmapResponse);
```

Comment:
why we cannot use the existing rpc `rpc update_delete_bitmap(UpdateDeleteBitmapRequest)`?

Reply:
update_delete_bitmap need to hold lock, this function no need to hold lock